rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -0,0 +1,137 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation/1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_openscap
+
+%global mainver 0.2.0
+%global release 1
+%{?prever:
+%global gem_instdir %{gem_dir}/gems/%{gem_name}-%{mainver}%{?prever}
+%global gem_docdir %{gem_dir}/doc/%{gem_name}-%{mainver}%{?prever}
+%global gem_cache %{gem_dir}/cache/%{gem_name}-%{mainver}%{?prever}.gem
+%global gem_spec %{gem_dir}/specifications/%{gem_name}-%{mainver}%{?prever}.gemspec
+}
+
+%define rubyabi 1.9.1
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: %{mainver}
+Release: 1%{?dist}
+Summary: Foreman plug-in for displaying OpenSCAP audit reports
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/OpenSCAP/foreman_openscap
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.5.0
+
+%if 0%{?fedora} > 18
+Requires: %{?scl_prefix}ruby(release)
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(deface)
+Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.2.0
```

Comment:
this dependency should be in the foreman_openscap gemspec, could you file/fix it there too please?